### ISSUE TITLE
Tests fail with "PyAsn1Error: Duplicate name notification"

### ISSUE THREAD

**Issue by vincentbernat** [etingof/pysmi#39]
Hey!

On Debian, tests are failing with the following error:

```
======================================================================
ERROR: testObjectTypeSyntax (test_objecttype_smiv2_pysnmp.ObjectTypeBitsTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/pysmi-0.3.1/tests/test_objecttype_smiv2_pysnmp.py", line 365, in testObjectTypeSyntax
    self.ctx['testObjectType'].getSyntax().clone(('set',)), str2octs('@'),
  File "/usr/lib/python2.7/dist-packages/pysnmp/proto/rfc1902.py", line 202, in clone
    return univ.OctetString.clone(self, *args, **kwargs).setFixedLength(self.getFixedLength())
  File "/usr/lib/python2.7/dist-packages/pyasn1/type/base.py", line 349, in clone
    return self.__class__(value, **initilaizers)
  File "/usr/lib/python2.7/dist-packages/pysnmp/proto/rfc1902.py", line 603, in __new__
    Bits = cls.withNamedBits(**kwargs.pop('namedValues'))
  File "/usr/lib/python2.7/dist-packages/pysnmp/proto/rfc1902.py", line 642, in withNamedBits
    class X(cls):
  File "/usr/lib/python2.7/dist-packages/pysnmp/proto/rfc1902.py", line 643, in X
    namedValues = cls.namedValues + namedval.NamedValues(*values.items())
  File "/usr/lib/python2.7/dist-packages/pyasn1/type/namedval.py", line 166, in __add__
    return self.__class__(*tuple(self.items()) + tuple(namedValues.items()))
  File "/usr/lib/python2.7/dist-packages/pyasn1/type/namedval.py", line 74, in __init__
    raise error.PyAsn1Error('Duplicate name %s' % (name,))
PyAsn1Error: Duplicate name notification

----------------------------------------------------------------------
Ran 108 tests in 6.123s
```

This happens with PySMI 0.3.1 and PyASN1 0.4.4. Any clue?

Thanks!

**Comment by etingof** [comment]
This seems to be a bug in pysnmp. Here is the [relevant issue](https://github.com/etingof/pysnmp/issues/193) raised.

**Comment by etingof** [comment]
Fix released in pysnmp 4.4.6.
